plinth/modules/pagekite/forms.py

```diff
@@ -18,12 +18,19 @@
 import copy
 from django import forms
 from django.contrib import messages
+from django.contrib.sites import requests
 from django.core import validators
+from django.core.exceptions import ValidationError
 from django.utils.translation import ugettext as _, ugettext_lazy
 import json
 import logging
 
-from plinth.errors import ActionError
+from plinth import cfg
+from plinth.errors import ActionError, DomainRegistrationError
+from plinth.modules.first_boot.forms import SubdomainWidget
```

Comment:
Consider moving SubdomainWidget over to pagekite module. I think it is only used by pagekite?